db/migrate/20201210183833_add_selected_domain_to_existent_properties.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class AddSelectedDomainToExistentProperties < ActiveRecord::Migration[6.0]
+  def change
+    Property.all.each do |p|
```

Comment:
This can be done at once, without looping through all of the properties:

`Property.update_all("selected_domain = domain->>0")`